libs/audio/src/module.ts

```diff
@@ -29,8 +29,11 @@ import {WebAudioMediaSource} from './sources/media-source';
 import {WebAudioMediaStreamSource} from './sources/media-stream-source';
 import {WebAudioOscillator} from './sources/oscillator';
 
+/**
+ * @deprecated: use standalone imports
```

Comment:
Let's not deprecate it, it's a pain in the ass to import them all one by one, let's keep module as an option